### ISSUE TITLE
Add Addtional Port Mappings

### ISSUE THREAD

**Issue by auddye** [clusterdock/topology_mapr#20]
Right now the only port mapping is 8443:8443, what about Drill UI and Zookeeper connectivity? can these be options?

**Comment by dimaspivak** [comment]
Yep, totally. Wanna put together a PR? I'll be happy to offer guidance.
